pkg/volume/validation/pv_validation.go

```diff
@@ -53,3 +57,15 @@ func checkMountOption(pv *api.PersistentVolume) field.ErrorList {
 	}
 	return allErrs
 }
+
+// ValidatePathNoBacksteps will make sure the targetPath does not have any element which is ".."
+func ValidatePathNoBacksteps(targetPath string) error {
+	parts := strings.Split(targetPath, string(os.PathSeparator))
```

Comment:
according to https://golang.org/src/os/path_windows.go?s=410:485#L1 windows accepts either `/` or `\`... not really sure what to do with that.

could do `strings.Split(filepath.ToSlash(targetPath), "/")`, I suppose